### DISCUSSION TITLE
Connect via FTP/SFTP in the container to get file access for mod support

### DISCUSSION THREAD

**Discussion by zVSciy** [wolveix/satisfactory-server#233] [Q&A]
So i wanted to use this container to play with friends, i found out that there is a way to get mods an a server but i need file access via FTP / SFTP but i dont know how the get username and password to get a connection. I tried to install a additional ssh server but the ssh server is stuck in a loop with the other config, so this does not work. PLS HELP

**Comment by wolveix** [comment]
Hey, this is unrelated to the container itself. It doesn't have an S/FTP server, and it doesn't need one. Map a directory to the host, and SFTP there :)
